tests/_files/test_animation.py

```diff
@@ -99,12 +101,31 @@ def test_to_dict(self, animation):
         assert animation_dict["file_unique_id"] == animation.file_unique_id
         assert animation_dict["width"] == animation.width
         assert animation_dict["height"] == animation.height
-        assert animation_dict["duration"] == animation.duration
+        assert animation_dict["duration"] == int(self.duration.total_seconds())
+        assert isinstance(animation_dict["duration"], int)
         assert animation_dict["thumbnail"] == animation.thumbnail.to_dict()
         assert animation_dict["file_name"] == animation.file_name
         assert animation_dict["mime_type"] == animation.mime_type
         assert animation_dict["file_size"] == animation.file_size
 
+    def test_time_period_properties(self, PTB_TIMEDELTA, animation):
```

Comment:
I don't think it's idiomatic to have an all caps fixture name in the argument. Maybe rename it?